### DISCUSSION TITLE
debug info - url quest api.opensea.io/v1/...

### DISCUSSION THREAD

**Discussion by max1404** [ProjectOpenSea/opensea-js#1193] [Q&A]
Hello,

I have updated opensea-js from version 5.0.3 to 6.1.9 a few days ago. With version 5.0.3 there was always a debug information on the API qeusts which URL / API endpoint is retrieved on api.opensea.io. This message does not come now. Is there any way to enable this again?

**Comment by ryanio** [comment]
Can you share an example of the debug information and what exactly you are looking for in it?

We changed the underlying HTTP request provider in v6 to ether's [fetchJson](https://docs.ethers.org/v5/api/utils/web/#utils-fetchJson). You should still get detailed information on any errors returned.

You could fork the library and add some logging before fetchJson is executed.

**Comment by max1404** [comment]
Before each API quest that was made, the cURL was output in the console.log:

`/v2/orders/ethereum/seaport/listings?limit=1&token_ids=xxxx&asset_contract_address=0x123456...&order_by=created_date&order_direction=desc`

It was just informative and I didn't find anything in the changelog.
